tests/codegen/sanitizer/aarch64-shadow-call-stack-with-fixed-x18.rs

```diff
@@ -0,0 +1,18 @@
+//@ needs-llvm-components: aarch64
+//@ revisions: aarch64 android
+//@[aarch64] compile-flags: --target aarch64-unknown-none -Zfixed-x18 -Zsanitizer=shadow-call-stack
+//@[android] compile-flags: --target aarch64-linux-android -Zfixed-x18 -Zsanitizer=shadow-call-stack
```

Comment:
The android one should also compile fine without `-Zfixed-x18`.

Reply:
True. Flag is dropped.